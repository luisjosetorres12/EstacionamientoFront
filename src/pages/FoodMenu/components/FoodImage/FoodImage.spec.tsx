import React from 'react';
import { render } from '@testing-library/react';
import { FoodImage } from '.';

describe('FoodImage Test', () => {
  it('should match snapshot', () => {
    const { container } = render(<FoodImage cover="food.jpg" />);
    expect(container).toMatchSnapshot();
  });
});
