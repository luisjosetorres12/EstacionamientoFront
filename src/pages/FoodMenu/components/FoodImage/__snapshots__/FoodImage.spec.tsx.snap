// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodImage Test should match snapshot 1`] = `
<div>
  <img
    alt="Food item"
    src="/images/food.jpg"
  />
</div>
`;
